iothub_client/tests/iothubtransport_mqtt_common_ut/iothubtransport_mqtt_common_ut.c

```diff
@@ -6488,7 +6523,57 @@ TEST_FUNCTION(IoTHubTransport_MQTT_Common_MessageRecv_with_Properties_succeed)
     IoTHubTransport_MQTT_Common_DoWork(handle);
     umock_c_reset_all_calls();
 
-    setup_message_recv_with_properties_mocks(true, true, false);
+    setup_message_recv_with_properties_mocks(true, true, false, true);
```

Comment:
nit: feel free to reject this one but it's difficult to parse what each of these represents without going to the function declaration. Consider having local vars for each which you set right above to make it easy to follow what each boolean is setting.

Reply:
here and elsewhere below

Reply:
I agree. Unfortunately that is disseminated in the C SDK tests, so I rather keep that as a separate task if we end up having time to document.